src/test/java/com/snack/news/service/CategoryServiceTest.java

```diff
@@ -4,66 +4,53 @@
-import static org.hamcrest.Matchers.equalTo;
-
-@RunWith(SpringRunner.class)
-@SpringBootTest
-public class CategoryServiceTest {
-
-	@Autowired
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class CategoryServiceTest {
+
+	@InjectMocks
 	private CategoryService categoryService;
 
-	@Autowired
+	@Mock
 	private CategoryRepository categoryRepository;
 
 	@Test
-	@Transactional
-	public void 카테고리를_생성할_수_있다() {
-		final String testCategoryTitle = Long.toString(ThreadLocalRandom.current().nextLong());
-		CategoryDto categoryDto = CategoryDto.builder().title(testCategoryTitle).build();
-		Category savedCategory = categoryService.createCategory(categoryDto);
+	@DisplayName("카테고리를 생성할 수 있다")
+	void createCategoryTest() {
+		CategoryDto categoryDto = CategoryDto.builder().title(anyString()).build();
 
-		Category expectedCategory = findCategoryByIdOrElseThrowException(savedCategory.getId());
-		assertThat(expectedCategory.getTitle(), equalTo(testCategoryTitle));
+		categoryService.createCategory(categoryDto);
+		verify(categoryRepository).save(any(Category.class));
 	}
 
 	@Test
-	@Transactional
-	public void 카테고리를_수정할_수_있다() {
-		final String originTestCategoryTitle = Long.toString(ThreadLocalRandom.current().nextLong());
-		CategoryDto originCategoryDto = CategoryDto.builder().title(originTestCategoryTitle).build();
-
-		Category savedCategory = categoryRepository.save(originCategoryDto.getNewEntity());
+	@DisplayName("카테고리를 수정할 수 있다")
+	void updateCategoryTest() {
+		CategoryDto originCategoryDto = CategoryDto.builder().build();
+		when(categoryRepository.existsById(any())).thenReturn(true);
 
-		final String updatedTestCategoryTitle = Long.toString(ThreadLocalRandom.current().nextLong());
-		CategoryDto updateCategoryDto = CategoryDto.builder().id(savedCategory.getId()).title(updatedTestCategoryTitle).build();
-
-		categoryService.updateCategory(updateCategoryDto);
-		Category expectedCategory = findCategoryByIdOrElseThrowException(savedCategory.getId());
-
-		assertThat(expectedCategory.getTitle(), equalTo(updatedTestCategoryTitle));
+		categoryService.updateCategory(originCategoryDto);
+		verify(categoryRepository).save(originCategoryDto.getUpdateEntity());
 	}
 
-	private Category findCategoryByIdOrElseThrowException(long id) {
-		return categoryRepository.findById(id).orElseThrow(CategoryNotFoundException::new);
-	}
-
-	@Test(expected = CategoryNotFoundException.class)
-	@Transactional
-	public void 카테고리_ID가_유효하지_않다면_예외를_반환한다() {
-		final long invalidId = ThreadLocalRandom.current().nextLong();
+	@Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4863700)